autoscale/src/main/java/com/sequenceiq/periscope/service/AltusMachineUserService.java

```diff
@@ -0,0 +1,98 @@
+package com.sequenceiq.periscope.service;
+
+import java.util.Optional;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.MachineUser;
+import com.google.common.collect.Multimap;
+import com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider;
+import com.sequenceiq.cloudbreak.auth.altus.GrpcUmsClient;
+import com.sequenceiq.cloudbreak.auth.altus.service.RoleCrnGenerator;
+import com.sequenceiq.cloudbreak.auth.crn.Crn;
+import com.sequenceiq.cloudbreak.logger.MDCUtils;
+import com.sequenceiq.freeipa.api.v1.freeipa.user.UserV1Endpoint;
+import com.sequenceiq.freeipa.api.v1.freeipa.user.model.SynchronizeAllUsersRequest;
+import com.sequenceiq.periscope.domain.Cluster;
+
+@Service
+public class AltusMachineUserService {
```

Comment:
The `AltusIAMService` in the auth-connector module is used by the other services' implementation of `AltusMachineUserService`(cloudbreak core and freeipa) which in turn invokes the UMS API via `GrpcUmsClient`. For periscope, `GrpcUmsClient` is used directly since the requirement is to create a machine user without access keys.